microsite/index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en" class="h-100">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="klio: smarter data pipelines for audio">
+    <meta name="docsearch:language" content="en" />
+    <title>klio: smarter data pipelines for audio</title>
+
+    <link rel="canonical" href="https://klio.io">
+    <link rel="shortcut icon" href="images/favicon.png"/>
+
+    <link href="css/bootstrap.css" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="./css/fonts.css" />
+    <link rel="stylesheet" type="text/css" href="./css/main.css" />
+  </head>
+
+  <body class="d-flex flex-column h-100 text-center">
+    <header>
+      <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
+        <a class="navbar-brand" href="#"><img src="images/logo.svg" class="logo" alt="logo"></a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+          <img src="images/menu.svg">
+        </button>
+        <div class="collapse navbar-collapse justify-content-end align-items-center" id="navbarCollapse">
+          <ul class="navbar-nav">
+            <li class="nav-item">
+              <a class="nav-link" href="https://github.com/spotify/klio"><button class="gradient">GitHub</button></a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="https://klio.readthedocs.io"><button class="gradient">Get Started</button></a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main role="main" class="flex-shrink-0">
+      <div class="shove-down"></div>
+      <div class="container mt-5">
```

Comment:
Wondering if it's possible to increase the height of this and/or add another shove-down div to increase the space between the wave image and the text here. Fine with not doing it if it's more complicated than that!

Reply:
@fallonchen That can be done, sure! I wasn't too concerned with keeping things _exact_ but if you think it looks better / is more readable, I'm happy to change it!